pom.xml

```diff
@@ -18,6 +18,7 @@
     <maven-compiler-plugin.version>3.5.1</maven-compiler-plugin.version>
     <maven-surefire-plugin.version>2.21.0</maven-surefire-plugin.version>
     <structured-logging.version>1.2.0-rc1</structured-logging.version>
+    <spring-framework.version>5.0.8.RELEASE</spring-framework.version>
 
```

Comment:
Are we using the correct version? cos everywhere else we declare the use of spring like so, 

See, company-accounts-api and document-generator 

<parent>
        <groupId>org.springframework.boot</groupId>
        <artifactId>spring-boot-starter-parent</artifactId>
        <version>2.0.3.RELEASE</version>
        <relativePath/> <!-- lookup parent from repository -->
    </parent>

       <dependency>
            <groupId>org.springframework.boot</groupId>
            <artifactId>spring-boot-starter-hateoas</artifactId>
            <scope>compile</scope>
        </dependency>